guides/common/modules/proc_reset-custom-ssl-certificate-to-default-self-signed-certificate-on-smartproxy.adoc

```diff
@@ -0,0 +1,43 @@
+[id="reset-custom-ssl-certificate-to-default-self-signed-certificate-on-{smart-proxy-context}_{context}"]
+= Reset {customssl} certificate to default self-signed certificate on {SmartProxyServer}
+
+Use this procedure to reset {customssl} certificate to default self-signed certificate on your {SmartProxyServer}.
+
+.On {ProjectServer}
+. Generate a new certificate tar bundle for {SmartProxyServer}.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {certs-generate} \
+--foreman-proxy-fqdn _{smart-proxy-context}.example.com_ \
+--certs-tar /root/_{smart-proxy-context}.example.com_-certs.tar.gz \
```

Comment:
The tar bundle generated [ or to be copied later on ] should not be `tar.gz` but just `tar` as far as I can tell. At least that is what every documentation points out. You can check out the smart-proxy\capsule installation doc for reference.

Reply:
Till now we had a `tar` but Ewoud fixed it with this [comment](https://bugzilla.redhat.com/show_bug.cgi?id=1874045#c13). I also see the same in this [PR](https://github.com/theforeman/foreman-installer/pull/911/files)

Reply:
It's only applicable from foreman 3.10 onwards. Sat 6.15 uses foreman 3.9 and hence the changes are not applicable there. Also, for a higher version i.e. 3.10, If you decide to mention `tar.gz` in one place, the same needs to be reflected in every other place as well.

Reply:
Noted. I can limit the cherry-picking of this PR up to `3.10` only. I will have another PR with just `tar` that will go beyond `3.9`.